.npmrc

```diff
@@ -1,17 +0,0 @@
-# This file defines default variables values used by npm scripts.
-# Read more at at:  https://docs.npmjs.com/files/npmrc
-# Config variables can be overridden by running:
-# npm run <script-name> --<variable-name>=<value>
-
-# General configuration
-engine-strict = true
-
-# Dashboard configuration
-kubeconfig_dir = ${HOME}/.kube
-kubeconfig = ${HOME}/.kube/config
-frontend_build_dir = dist/frontend/en
-dashboard_binary_path = dist/amd64/dashboard
-dashboard_locale_config = dist/amd64/locale_conf.json
-bind_address = 127.0.0.1
-sidecar_host = http://localhost:8000
-port = 8080
```

Comment:
I use these params. Could you enable to use them in new scripts?

Reply:
See the top of the makefile. You can still specify them, i.e. "ENV_VAR=test make test".

Reply:
Okay, I see now. Port and address are missing. I'll revert .npmrc file.